### ISSUE TITLE
Won't run with recent version of MacFuse (workaround)

### ISSUE THREAD

**Issue by awd97** [ivoronin/ArchiveMounter#8]
as the MacFuse executable has changed name from 'osxfuse' to 'macfuse'. A workaround is to copy /Library/Filesystems/macfuse.fs to /Library/Filesystems/osxfuse.fs and then rename /Library/Filesystems/macfuse.fs/Contents/Resources/mount_osxfuse to /Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse

**Comment by LewsTherinTelescope** [comment]
Thanks for that workaround! Symlinking things worked perfectly to fix it.

**Comment by lukebrowell** [comment]
AppDelegate.swift needs updating to reflect new fuse name:
"/Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse"

**Comment by ivoronin** [comment]
Pull requests are welcome!

**Comment by MorningSage** [comment]
> /Library/Filesystems/macfuse.fs/Contents/Resources/mount_osxfuse to /Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse

Vice verse...  But thank you, this saved me.  Definitely gets things working.

**Comment by LeeBinder** [comment]
> rename /Library/Filesystems/macfuse.fs/Contents/Resources/mount_osxfuse to /Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse

No, mistake. Instead the other way around. Also symlinking should definitely be used instead to leave MacFuse as is, and it does work:

symlink `/Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse` to `/Library/Filesystems/macfuse.fs/Contents/Resources/mount_osxfuse`

and the step before

symlink `/Library/Filesystems/macfuse.fs` to `/Library/Filesystems/osxfuse.fs`

![Bildschirmfoto 2022-12-01 um 21 23 33](https://user-images.githubusercontent.com/39203497/205152210-216861a0-fffd-43a0-974c-13841c33d722.jpg)

**Comment by BenK885** [comment]
> symlink `/Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse` to `/Library/Filesystems/macfuse.fs/Contents/Resources/mount_osxfuse`

`sudo ln -s /Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse /Library/Filesystems/macfuse.fs/Contents/Resources/mount_osxfuse`

 
> symlink `/Library/Filesystems/macfuse.fs` to `/Library/Filesystems/osxfuse.fs`

`sudo ln -s /Library/Filesystems/macfuse.fs /Library/Filesystems/osxfuse.fs`
